### PR TITLE
Added config:update-php-version command.

### PR DESCRIPTION
**Pull request by markdorison**
## Description
[Added config:update-php-version command.](https://github.com/ChromaticHQ/usher/commit/ed28da50cf812f15cde64a17b31f735e504f42ae)

## Motivation / Context
Idea while working on Robo talk.

## Testing Instructions / How This Has Been Tested
Tested locally.

### CODE REVIEW

File: src/Robo/Plugin/Commands/ToolingCommands.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                ->to($version)
+                ->run();
+
+            if (str_contains($configPath, $composerFilename)) {
+                $this->say("Change to $composerFilename detected.");
+                if ($opts['skip-composer-update']) {
+                    $this->yell("'composer update' skipped.");
+                    continue;
+                }
+                $this->yell('Updating composer.lock file.');
```
Comment (adamzimmermann) [inline src/Robo/Plugin/Commands/ToolingCommands.php:81]: 👏🏼

File: src/Robo/Plugin/Commands/ToolingCommands.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+     *   The key to update.
+     * @param string $value
+     *   The value to set it to.
+     */
+    protected function updateRoboConfig(string $key, string $value): void
+    {
+        $roboConfigPath = "$this->cwd/robo.yml";
+        $roboConfig = Yaml::parse(file_get_contents($roboConfigPath));
+        $roboConfig[$key] = $value;
+        file_put_contents($roboConfigPath, Yaml::dump($roboConfig));
```
Comment (adamzimmermann) [inline src/Robo/Plugin/Commands/ToolingCommands.php:107]: Does this mess up the formatting like we have seen in the past?

**Review by adamzimmermann — APPROVED** [review]
I like it!

File: src/Robo/Plugin/Commands/ToolingCommands.php (hunk 2 above)
Comment (markdorison) [reply src/Robo/Plugin/Commands/ToolingCommands.php:107]: In my testing, it only stripped quotes (that aren't needed). We don't have any comments in the file I tested on so no problem there.

**Pull request merged**